dashboard/postgres_replicator/logger_test.go

```diff
@@ -0,0 +1,36 @@
+package transfer
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+	"testing"
+)
+
+func TestNewLogger(t *testing.T) {
+	tableName := "foobar"
+	logger := NewLogger(tableName)
+	expectedPrefix := fmt.Sprintf("[%s] ", tableName)
```

Comment:
Tests should typically not have logic to calculate expected values, because that could replicate the action of the original code. I would also use "got" and "want" for expected and actual.

You may find this package useful in general: https://pkg.go.dev/github.com/google/go-cmp/cmp

Reply:
Changed!